client/src/components/CarouselEntry.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CarouselEntry = ({ listing }) => (
```

Comment:
I really like this destructuring I wonder if I can do that with Component classes